ext/storage_sources/s3_store/s3_log_system.h

```diff
@@ -43,7 +43,11 @@ static const std::map<int32_t, Aws::Utils::Logging::LogLevel> verbosityMapping =
   {WT_VERBOSE_WARNING, Aws::Utils::Logging::LogLevel::Error},
   {WT_VERBOSE_WARNING, Aws::Utils::Logging::LogLevel::Warn},
   {WT_VERBOSE_INFO, Aws::Utils::Logging::LogLevel::Info},
-  {WT_VERBOSE_DEBUG, Aws::Utils::Logging::LogLevel::Debug}};
+  {WT_VERBOSE_DEBUG_1, Aws::Utils::Logging::LogLevel::Debug},
+  {WT_VERBOSE_DEBUG_2, Aws::Utils::Logging::LogLevel::Debug},
+  {WT_VERBOSE_DEBUG_3, Aws::Utils::Logging::LogLevel::Debug},
```

Comment:
How did we decide that this is the right way to do this?

Reply:
It needs to change from debug to trace at some point - I don't think there's a single "correct" point, so I picked `DEBUG_4`.

Reply:
I think my preference would be for anything which is more verbose than debug_1 to be "trace". Then if we have to make similar decisions elsewhere we can be consistent. What do you think?

Reply:
That's a reasonable position. My understanding (from some Googling, so not especially well-founded) is that trace is extremely verbose, so I thought a higher level would be appropriate. I'll ask Ruby for some input too.

Reply:
I agree with Will's current position. As I came to understand during the S3 project, Trace is extremely verbose and provides a lot of information that I do not think would be necessary for many debugging purposes and would flood the logs. I think keeping trace starting at WT_VERBOSE_DEBUG_4 or even only at WT_VERBOSE_DEBUG_5 at the last level of verbosity is a good call.

Reply:
Thanks for the feedback @ruby-oujo ! I've changed it so that only DEBUG_5 maps to the Trace level.

Reply:
Sounds good.